README.md

```diff
@@ -1,15 +1,46 @@
-# Project Name
+# High Performance Vision-Guided Camera Tracker  
 
-Developer Names:
+**Developer Names**: Zifan Si, Jianqing Liu, Mike Chen, Xiaotian Lou  
+**Supervisor**: TBD  
 
-Date of project start:
+Date of project start: September 8th 2025  
 
-This project is ...
+**Project Overview**: The goal of this project is to design and implement a deployable camera tracking system capable of autonomously locking onto and following extremely fast-moving small targets, such as model rockets. The system integrates computer vision, motion control, and a web-based operator interface to deliver smooth, stable video (1080p @ 60fps) of critical rocket events such as staging and parachute deployment.  
 
-The folders and files for this project are as follows:
+This project builds a software stack that interfaces with low-cost hardware for small-scale launches (200m apogee) and is scalable to support the McMaster Rocketry Team’s high-powered rockets (3km+ apogee).  
 
-docs - Documentation for the project
-refs - Reference material used for the project, including papers
-src - Source code
-test - Test cases
-etc.
\ No newline at end of file
+---
+
+### **Key Features**  
```

Comment:
Key Features should not include implmentation details (e.g. STM32 + Rust, or React + Flask, or Jetson